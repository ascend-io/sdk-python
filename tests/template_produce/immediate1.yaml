id: immediate1
component:
  readConnector:
    container:
      immediate:
        content_some: RXhhbXBsZSB0ZXh0IGJsb2IgZmlsZSBwcm9kdWN0aW9uCkxpbmUgMiBvZiBFeGFtcGxlIHRleHQgYmxvYiBmaWxlIHByb2R1Y3Rpb24KVGhpcyBibG9iIGhhcyBqaW5qYSB7eyBjaGFycyB9fSBpbiBpdAo=
        object:
          - name: ascend_text_blob
            length: 116
    updatePeriodical:
      offset: 103366736s
      period: 134217727s
