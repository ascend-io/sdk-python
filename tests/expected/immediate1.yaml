# id: immediate1
{%- set ascendImmediateDir -%}{{ resourceParentDirectory }}/immediate1{%- endset %}
{%- set ascendImmediateFiles = ["ascend_text_blob"] %}
component:
  readConnector:
    container:
      immediate: {{ construct_immediate_container(ascendImmediateDir, ascendImmediateFiles) }}
    updatePeriodical:
      offset: 103366736s
      period: 134217727s
